File: tasks/prac2_js/task8.js
const prompt = require('prompt-sync')();

let zena = parseFloat(prompt("Введите исходную цену: "));
let skidka = parseFloat(prompt("Введите процент скидки: "));

let novaya_zena = zena * (1 - skidka / 100);
console.log(`Цена после скидки: ${novaya_zena}`);

samaya_novaya_zena = novaya_zena - 10;
console.log(`Цена после купона: ${samaya_novaya_zena}`);

File: tasks/prac2/task8.py
zena = int(input("Введите цену: "))
skidka = int(input("Введите процент скидки: "))

novaya_zena = zena * (1 - skidka / 100)
print(f"Цена после скидки: {novaya_zena}")

samaya_novaya_zena = novaya_zena - 10
print(f"Цена после купона {samaya_novaya_zena}")
